pkg/controller/ingress/ingress_controller.go

```diff
@@ -184,13 +188,13 @@ func (r *ReconcileIngress) fetchNetworkingInfo(instance *extensionsv1beta1.Ingre
 		nodeInstanceIds = append(nodeInstanceIds, node.Spec.ProviderID[strings.LastIndex(node.Spec.ProviderID, "/")+1:])
 	}
 
-	r.log.Info("getting vpcID, securityGroups, subnetIds for worker nodes")
-	vpcIDs, subnetIds, securityGroups, err := network.GetNetworkInfoForEC2Instances(r.ec2Svc, nodeInstanceIds)
+	r.log.Info("getting vpcID, securityGroups, subnetIds, asgNames for worker nodes")
+	vpcIDs, subnetIds, securityGroups, asgNames, err := network.GetNetworkInfoForEC2Instances(r.ec2Svc, r.autoscalingSvc, nodeInstanceIds)
 	if err != nil {
 		return nil, err
 	}
 
-	r.log.Info("creating API Gateway cloudformation stack")
+	r.log.Info("describing VPCs")
```

Comment:
Damn the previous log msg makes absolutely no sense 🤦‍♂ , That said can we change this to better reflect the work being done - (Checking if the VPCId : %s exists). 

Minor : Also this check seems redundant, we are getting the VPC Id from the instance, shouldn't that guarantee that the VPC exists

Reply:
Fixing log message.

I think in next line it is making sure the VPC exists.